### PR TITLE
Support parallel rebalance in cli operations

### PR DESCRIPTION
**Pull request by antban**
ARUHA-805
Right now bubuku makes rebalance in step by step manner. Instead of that one can balance many partitions at a time, and it should be much faster.
The goal of this PR is to support step size for parallel rebalance

### CODE REVIEW

**Comment by antban** [comment]
:+1:

File: bubuku/cli.py [hunk 1]
```diff
@@ -85,15 +85,17 @@ def restart_broker(broker: str):
 @click.option('--empty_brokers', type=click.STRING,
               help="Comma-separated list of brokers to empty. All partitions will be moved to other brokers")
 @click.option('--exclude_topics', type=click.STRING, help="Comma-separated list of topics to exclude from rebalance")
-def rebalance_partitions(broker: str, empty_brokers: str, exclude_topics: str):
+@click.option('--step', type=click.INT, show_default=True,
```
Comment (antban) [inline bubuku/cli.py]: @lmontrieux Yep, you are completely right here. Therm step is completely wrong here, I will rename it everywhere to parallelism

File: bubuku/cli.py [hunk 2]
```diff
@@ -85,15 +85,17 @@ def restart_broker(broker: str):
...
+              help="Step size for rebalance, i.e. how many parallel moves do at a time")
+def rebalance_partitions(broker: str, empty_brokers: str, exclude_topics: str, step: int = 1):
```
Comment (antban) [inline bubuku/cli.py]: The check is a little bit deeper - in process of command creation for rebalance

**Comment by antban** [comment]
@lmontrieux I fixed everything but additional check in cli, cause parallelism is checked inside command creation: https://github.com/zalando-incubator/bubuku/pull/88/commits/b8f937ab77b390162868bcf7a4b72396e3b48ce3

File: bubuku/cli.py [hunk 3]
```diff
@@ -106,14 +106,14 @@ def rebalance_partitions(broker: str, empty_brokers: str, exclude_topics: str, s
 @click.option('--shrink', is_flag=True, default=False, show_default=True,
               help='Whether or not to shrink replaced broker ids form partition assignment')
 @click.option('--broker', type=click.STRING, help='Optional broker id to execute check on')
-@click.option('--step', type=click.INT, show_default=True,
-              help="Step size for rebalance, i.e. how many parallel moves do at a time")
-def migrate_broker(from_: str, to: str, shrink: bool, broker: str, step: int = 1):
+@click.option('--parallelism', type=click.INT, show_default=True,
+              help="Amount of partitions to move in a single rebalance step")
```
Comment (lmontrieux) [inline bubuku/cli.py]: `rebalance` -> `migration`

**Comment by lmontrieux** [comment]
@antban I'm almost ready to approve it :)

**Comment by lmontrieux** [comment]
👍

**Comment by lmontrieux** [comment]
👍

**Comment by antban** [comment]
:+1:

**Comment by antban** [comment]
:+1:

**Comment by lmontrieux** [comment]
👍

**Pull request merged**